redeyed.js

```diff
@@ -188,6 +186,12 @@ function bootstrap(esprima, exportFn) {
       , info
       ;
 
+    // Replace hashbang line with empty whitespaces to preserve token locations
+    if (code[0] === '#' && code[1] === '!') {
```

Comment:
Yes. 99% of the time, there is no hashbang so the first check will bypass the entire block right away.
Beside, we don't want to replace the string so there is no need to regex.